hs_core/tests/data/readme.txt

```diff
@@ -0,0 +1 @@
+readme
```

Comment:
Is this a placeholder readme file or some brief real readme text needs to be put in the file?

Reply:
The tests are relevant to the file name only (`readme.txt`) and not the content of this file.